src/opm/parser/eclipse/Parser/Parser.cpp

```diff
@@ -660,6 +660,15 @@ bool parseState( ParserState& parserState, const Parser& parser ) {
         if( parser.isRecognizedKeyword( parserState.rawKeyword->getKeywordName() ) ) {
             const auto& kwname = parserState.rawKeyword->getKeywordName();
             const auto* parserKeyword = parser.getParserKeywordFromDeckName( kwname );
+            {
+                const auto& rawKeyword = *parserState.rawKeyword;
+                std::stringstream ss;
+
+                ss << std::setw(5) << parserState.deck.size()
+                   << " Reading " << std::setw(8) << std::left << rawKeyword.getKeywordName()
```

Comment:
The setw(8) should be redundant, but I guess better safe than sorry?

This logging is done unconditionally, I guess it should be activated with a switch in some way?

Also, there has in the past been some cases where you did not get log messages because the logging system was not initialized yet, I think that is ok now?

Reply:
> The setw(8) should be redundant, but I guess better safe than sorry?

The intention was to get left adjusted, and space padded to eight characters. I guess the `setw()` is required to achieve that?

Reply:
>This logging is done unconditionally, I guess it should be activated with a switch in some way?

Well - isn't that that the purpose of the logging system; which can decide to drop these messages on the floor?

Reply:
> Also, there has in the past been some cases where you did not get log messages because the logging system was not initialized yet, I think that is ok now?

I would say so; if the logging system has not been initiialized the messages will be droppped - and then that is OK.